test/ssl_test_ctx.c

```diff
@@ -154,6 +154,62 @@ const char *ssl_verify_callback_name(ssl_verify_callback_t callback)
                      callback);
 }
 
+/*************************/
+/* SessionTicketExpected */
```

Comment:
Should say ServerName.

Could you also update README.ssltest.md with a brief mention of these new options - there's a section there that lists all known options. And add tests to ssl_test_ctx_test.c / ssl_test_ctx_test.conf (add one good option to the "good" test and one invalid option test per option, see examples there, should be quick).